src/test/java/org/apache/sling/jcr/maintenance/internal/VersionCleanupTest.java

```diff
@@ -133,13 +153,32 @@ public void testRunnable() throws InterruptedException, VersionException, Unsupp
         assertEquals(5L, vcs.getLastCleanedVersionsCount());
     }
 
+    @Test(timeout = 5000)
```

Comment:
Why do we need this timeout?

Reply:
You're right, that it's unnecessary in this scenario